specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -1097,17 +1097,1048 @@
+          }
+        }
+      },
+      "patch": {
+        "description": "Update a persisted face's userData field.",
+        "operationId": "LargeFaceList_UpdateFace",
+        "parameters": [
+          {
+            "$ref": "#/parameters/largeFaceListId"
+          },
+          {
+            "$ref": "#/parameters/persistedFaceId"
+          },
+          {
+            "name": "body",
+            "description": "Request body for updating persisted face.",
+            "in": "body",
+            "required": true,
+            "x-ms-client-flatten": true,
+            "schema": {
+              "$ref": "#/definitions/UpdateFaceRequest"
+            }
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "A successful call returns an empty response body."
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Update face example": {
+            "$ref": "./examples/UpdateLargeFaceListFace.json"
+          }
+        }
+      }
+    },
+    "/largefacelists/{largeFaceListId}/persistedfaces": {
+      "post": {
+        "description": "Add a face to a large face list. The input face is specified as an image with a targetFace rectangle. It returns a persistedFaceId representing the added face, and persistedFaceId will not expire.",
+        "operationId": "LargeFaceList_AddFaceFromUrl",
+        "parameters": [
+          {
+            "$ref": "#/parameters/largeFaceListId"
+          },
+          {
+            "$ref": "#/parameters/faceUserData"
+          },
+          {
+            "$ref": "#/parameters/targetFace"
+          },
+          {
+            "$ref": "../../../Common/Parameters.json#/parameters/ImageUrl"
+          }
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "A successful call returns a new persistedFaceId.",
+            "schema": {
+              "$ref": "#/definitions/PersistedFace"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create large face list face example": {
+            "$ref": "./examples/AddLargeFaceListFaceFromUrl.json"
+          }
+        }
+      },
+      "get": {
+        "description": "List all faces in a large face list, and retrieve face information (including userData and persistedFaceIds of registered faces of the face).",
+        "operationId": "LargeFaceListFace_List",
```

Comment:
I think this operation ID should be changed to "LargeFaceList_ListFaces"
This would make it more consistent with the other operations like "LargeFaceList_GetFace" and "LargeFaceList_DeleteFace" etc.

Alternatively you could rename all of them: "LargeFaceListFace_Delete", "LargeFaceListFace_Get" etc

Reply:
@annatisch Also updated our generated .NET [SDK ](https://github.com/lebronJ/azure-sdk-for-net/pull/2) and all works.